_migrations/getting-started-data-migration.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-title: Quickstart: Data migration
```

Comment:
Alternatively, you can just put the title in quotation marks.

Reply:
Good to know!